src/nvim/version.c

```diff
@@ -57,7 +57,6 @@ static char *(features[]) = {
   "+cursorbind",
   "+cursorshape",
   "+dialog_con",
-  "+diff",
```

Comment:
I don't think this should be removed, some plugin code might want to check on `has('diff')`.